packages/web-components/src/components/ic-menu-group/ic-menu-group.tsx

```diff
@@ -17,13 +17,13 @@ export class MenuGroup {
     const parentMenu = this.el.closest("ic-popover-menu");
 
     return (
-      <Host aria-label={this.label !== null ? this.label : ""}>
+      <Host role="group" aria-label={this.label !== null ? this.label : ""}>
         {isPropDefined(this.label) && (
           <ic-typography variant="subtitle-small">{this.label}</ic-typography>
         )}
-        <ul role="group">
+        <span>
```

Comment:
Does moving the group role have a negative impact on accessibility? Menu items use the <li> tag and they need to be placed in either a group or a list

Reply:
Because of your comment I took a second look at the file and have refactored it a little bit more - previously the group role being nested on the ul was causing an error, having it on the host element of the ic-menu-group removes the error and functions fine in voiceover :) 
<img width="622" alt="Screenshot 2024-11-08 at 11 10 41" src="https://github.com/user-attachments/assets/af1baa81-b1fd-48ba-8eaa-7d81b203f94e">